static/app/views/starfish/views/screens/index.tsx

```diff
@@ -434,3 +374,95 @@ export const Spacer = styled('div')`
 const StyledSearchBar = styled(SearchBar)`
   margin-bottom: ${space(1)};
 `;
+
+function getSetupContent(): CodeSnippetTab[] {
```

Comment:
might be nice to have these snippets stored as constants in a separate file somewhere